config/topic_config.lua

```diff
@@ -0,0 +1,23 @@
+-- define multi robot senerio
+robot_number = 2
+
+-- topic prefix of robots
+topic_prefix_list = {"car0", "car1", "car2"}
```

Comment:
This information (and a lot of the other information in this file) seem to be repeated from sim_config.lua. Is there a reason why this is duplicated, and is there a way it could not be?

Reply:
Sorry, my intention was to move all variables in this file to sim_config. I was going to remove this file but forgot about doing this while I was merging the commits.